packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -34,6 +56,21 @@ Future<List<XFile>> openFiles({
 }
 
 /// Saves File to user's file system
+///
+/// The [acceptedTypeGroups] argument is the file type that can be selected in the dialog.
+/// When omitted, Open file dialog with all file types.
+///
+/// The [initialDirectory] argument is directory that will be displayed when the dialog is opened.
+/// (NOTICE: specify a directory as a full path, not a relative path.)
+/// When omitted, The result of each platform's `getDirectoryPath()` execution is used.
+///
+/// The [suggestedName] argument is he name of the file to save.
```

Comment:
s/he/the/

This also makes it sound like it forces a name. It's actually just the initial value of the name.